### ISSUE TITLE
Trimou not compatible with Jakarta 10

### ISSUE THREAD

**Issue by RoopamGupta** [trimou/trimou#142]
I am facing issue with Trimou compatibility with JavaEE 10.
We are working to upgrade wildfly to latest where it has updated from JavaEE 8 to JavaEE 10 so we need to upgrade from Javax -> Jakarta package but i am seeing that Trimou is not compatible with the Jakarta while loading swagger template using servlet context.

TemplateLocator locator = ServletContextTemplateLocator.builder().setServletContext( servletContext ).build(); -> Throwing compilation Issue

**Comment by RoopamGupta** [comment]
Error Logged : org.trimou.exception.MustacheException: TEMPLATE_LOADING_ERROR: Servlet context not available

**Comment by mkouba** [comment]
Yes, Trimou extensions are not Jakarta EE 10 compatible. However, the core should not depend on any class from the `javax` package. So you can fork the repo and create a Jakarta-compatible `trimou-extension-servlet` dependency easily. 

Pull requests are also welcome ;-)

**Comment by mkouba** [comment]
I will not have time to work on this in the near future.

But let's set the milestone for this issue and also the tentative due date for the [3.0.0](https://github.com/trimou/trimou/milestone/29).

**Comment by RoopamGupta** [comment]
Thanks for response.

**Comment by RoopamGupta** [comment]
@mkouba : Did we make any progress on this ?

**Comment by RoopamGupta** [comment]
@mkouba : Awaiting for your valuable response. Reaching out to you as i got stuck with some urgent work which is requiring this stuff.

**Comment by RoopamGupta** [comment]
@mkouba : Any update?

**Comment by mkouba** [comment]
This issue was added to the [3.0.0](https://github.com/trimou/trimou/milestone/29) milestone which is due by September 29 (at least that's the plan).

**Comment by RoopamGupta** [comment]
Hi @mkouba - Did we make any progress on this? As this was supposed to be released 3.0.0 at the end of September.

**Comment by RoopamGupta** [comment]
Any further update on this?

**Comment by mkouba** [comment]
It's work in progress ;-)

**Comment by RoopamGupta** [comment]
Thats good news, Any tentative plan for release?

**Comment by mkouba** [comment]
As fast as it goes...

The WIP branch is here: https://github.com/mkouba/trimou/tree/jakarta-ee-10
